packages/main/src/Tokenizer.ts

```diff
@@ -275,14 +277,14 @@ class Tokenizer extends UI5Element {
 		}
 	}
 
-	_tokenKeyboardDelete(event, token) {
+	_tokenKeyboardDelete(e: CustomEvent, token: Token) {
```

Comment:
if this uses the delete event from token use the newly created type

Reply:
Done